Assets/Scripts/UI/PickupPup/DogSlot/DogCollarSlot.cs

```diff
@@ -1,10 +1,13 @@
 ﻿/*
- * Author: Grace Barrett-Snyder 
+ * Authors: Grace Barrett-Snyder, Ben Page
  * Description: Controls a DogSlot for a Dog that's outside (has name and timer).
  */
 
 using UnityEngine;
 using UnityEngine.UI;
+using System;
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
System.Collections for the IEnumerator I added.
Wasn't using '.Generic', taken care of.